pkg/tenant/projects.go

```diff
@@ -175,11 +175,23 @@ func (s *Server) ProjectList(c *gin.Context) {
 func (s *Server) ProjectCreate(c *gin.Context) {
 	var (
 		err     error
+		p       *tokens.Claims
 		project *api.Project
-		out     *api.Project
 	)
 
-	// TODO: Add authentication middleware to fetch the organization ID.
+	// Fetch project from the context.
+	if p, err = middleware.GetClaims(c); err != nil {
+		log.Error().Err(err).Msg("could not fetch project from context")
+		c.JSON(http.StatusUnauthorized, api.ErrorResponse(err))
+		return
+	}
+
+	// Get the project's organization ID and return a 500 response if it is not a ULID.
```

Comment:
The Project model takes in a Tenant ID, not an OrgID and that is why the value isn't captured here and used elsewhere.